internal/cli/cmd/kubeblocks/data/eks_hostpreflight.yaml

```diff
@@ -36,8 +36,12 @@ spec:
         regionNames:
           - cn-northwest-1
           - cn-north-1
+          - us-east-1
+          - us-east-2
+          - us-west-1
+          - us-west-2
         outcomes:
           - warn:
-              message: k8s cluster region doesn't belong to amazon china
+              message: k8s cluster region doesn't belong to amazon china/us, you may have some unexpected behavior
```

Comment:
k8s cluster region doesn't belong to amazon china/us, you may have some unexpected behavior -> be aware of the network reachability for cluster regions that located at East-Asia